App/src/services/auth.ts

```diff
@@ -8,6 +9,21 @@ export interface AuthenticationResponse {
     };
 }
 
+export interface DecodedToken {
```

Comment:
this is a duplicate interface. it already exists in services/user.ts

Reply:
It's not duplicated. services/user.ts contains a User interface. The DecodedToken interface was moved from the Login page to services/auth.ts since it's being used in multiple spots now.

Reply:
you are right